pkg/envoy/ads/stream.go

```diff
@@ -20,6 +22,42 @@ import (
 	"github.com/openservicemesh/osm/pkg/utils"
 )
 
+// This is the workqueue Job implementation to issue a Send Response to a proxy
+type sendJob struct {
+	// Send Response parametres
+	typeurls  mapset.Set
```

Comment:
Wouldn't it be simpler if we used a `[]TypeURL` here?  What characteristics of the `mapset` do we need?
It seems to me that if we used `[]TypeURL` we'd preserve the type and it carries more semantic context around what this is.

Reply:
mapset prevents issuing or requesting updates multiple times for the same typeurl. It prevents user error; a slice doesn't